essential-tests/build.gradle.kts

```diff
@@ -32,6 +30,8 @@ dependencies {
   testFixturesImplementation(project(":service-runner"))
   testFixturesImplementation(project(":lib-util"))
 
+  testFixturesImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test")
```

Comment:
Is this necessary? If yes, can we move it to the `libs.version.toml`?

Reply:
No, I forgot that I added it :)